docs/detections/prebuilt-rules/prebuilt-rules-reference.asciidoc

```diff
@@ -668,10 +684,12 @@ and their rule type is `machine_learning`.
 
 |<<nullsessionpipe-registry-modification, NullSessionPipe Registry Modification>> |Identifies NullSessionPipe registry modifications that specify which pipes can be accessed anonymously. This could be indicative of adversary lateral movement preparation by making the added pipe available to everyone. |[Elastic] [Host] [Windows] [Threat Detection] [Lateral Movement]  |7.13.0 |1
 
-|<<o365-excessive-single-sign-on-logon-errors, O365 Excessive Single Sign-On Logon Errors>> |Identifies accounts with a high number of single sign-on (SSO) logon errors. Excessive logon errors may indicate an attempt to brute force a password or SSO token. |[Elastic] [Cloud] [Microsoft 365] [Continuous Monitoring] [SecOps] [Identity and Access]  |7.14.0 |2 <<o365-excessive-single-sign-on-logon-errors-history, Version history>>
+|<<o365-excessive-single-sign-on-logon-errors, O365 Excessive Single Sign-On Logon Errors>> |Identifies accounts with a high number of single sign-on (SSO) logon errors. Excessive logon errors may indicate an attempt to brute force a password or SSO token. |[Elastic] [Cloud] [Microsoft 365] [Continuous Monitoring] [SecOps] [Identity and Access]  |7.14.0 |3 <<o365-excessive-single-sign-on-logon-errors-history, Version history>>
 
 |<<o365-exchange-suspicious-mailbox-right-delegation, O365 Exchange Suspicious Mailbox Right Delegation>> |Identifies the assignment of rights to accesss content from another mailbox. An adversary may use the compromised account to send messages to other accounts in the network of the target business while creating inbox rules, so messages can evade spam/phishing detection mechanisms. |[Elastic] [Cloud] [Microsoft 365] [Continuous Monitoring] [SecOps] [Configuration Audit]  |7.16.0 |1
 
+|<<o365-mailbox-audit-logging-bypass, O365 Mailbox Audit Logging Bypass>> |Detects the occurrence of mailbox audit bypass associations. The mailbox audit is responsible for logging specified mailbox events (like accessing a folder or a message or permanently deleting a message). However, actions taken by some authorized accounts, such as accounts used by third-party tools or accounts used for lawful monitoring, can create a large number of mailbox audit log entries and may not be of interest to your organization. Because of this, administrators can create bypass associations, allowing certain accounts to perform their tasks without being logged. Attackers can abuse this allowlist mechanism to conceal actions taken, as the mailbox audit will log no activity done by the account. |[Elastic] [Cloud] [Microsoft 365] [Continuous Monitoring] [SecOps] [Initial Access]  |8.0 |1
```

Comment:
```suggestion
|<<o365-mailbox-audit-logging-bypass, O365 Mailbox Audit Logging Bypass>> |Detects the occurrence of mailbox audit bypass associations. The mailbox audit is responsible for logging specified mailbox events (like accessing a folder or a message or permanently deleting a message). However, actions taken by some authorized accounts, such as accounts used by third-party tools or accounts used for lawful monitoring, can create a large number of mailbox audit log entries and may not be of interest to your organization. Because of this, administrators can create bypass associations, allowing certain accounts to perform their tasks without being logged. Attackers can abuse this allowlist mechanism to conceal actions taken, as the mailbox audit will log no activity done by the account. |[Elastic] [Cloud] [Microsoft 365] [Continuous Monitoring] [SecOps] [Initial Access]  |8.0.0 |1
```